src/routes/orders/create-order.js

```diff
@@ -0,0 +1,64 @@
+const orderParent = require('../../modules/db/schemes/order-scheme.js');
+const productParent = require('../../modules/db/schemes/product-scheme.js');
+const userParent = require("../../modules/db/schemes/userScheme");
+
+const createOrder = req => {
+    const order = req.body;
+
+    return new orderParent({
+        ...order
+    });
+};
+
+const saveOrder = (req, res) => {
+    const newOrder = createOrder(req);
+
+    const saveOrderFromUser = async order => {
+        const userId = order.creator;
+
+        return userParent.findById(userId)
+            .then(user => {
```

Comment:
use async await instead of then in all project

Reply:
all async/await constructions should be wrapped on try/catch

example https://miro.medium.com/max/1800/0*W87pvlVeQVGyaVO0